Formula/chkrootkit.rb

```diff
@@ -1,9 +1,8 @@
 class Chkrootkit < Formula
   desc "Rootkit detector"
   homepage "http://www.chkrootkit.org/"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/c/chkrootkit/chkrootkit_0.50.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/c/chkrootkit/chkrootkit_0.50.orig.tar.gz"
```

Comment:
I was able to find a mirror, odd that it's not in Debian yet. Arch, Gentoo, and FreeBSD all pull from that original FTP for their ports, I was able to find a mirror, with the correct md5 and had successful install, it has been added to the formula.